src/asm/x86/mc.rs

```diff
@@ -363,3 +363,88 @@ pub(crate) static AVG_FNS: [Option<AvgFn>; CpuFeatureLevel::len()] = {
 
 pub(crate) static AVG_HBD_FNS: [Option<AvgHBDFn>; CpuFeatureLevel::len()] =
   [None; CpuFeatureLevel::len()];
+
+#[rustfmt::skip]
+static MC_SUBPEL_FILTERS: AlignedArray<[i8; 600]> = AlignedArray::new([
```

Comment:
The following change passes tests for me with feature `decode_test_dav1d`:
```suggestion
#[no_mangle]
pub static rav1e_mc_subpel_filters: AlignedArray<[i8; 600]> = AlignedArray::new([
```

Reply:
Right, that was my doubt, I still do not understand how it could work for the aom tests though.